packages/nimble-components/src/combobox/testing/combobox.pageobject.ts

```diff
@@ -0,0 +1,183 @@
+import { keyEnter, keyArrowDown } from '@microsoft/fast-web-utilities';
+import type { Combobox } from '..';
+import { listOptionTag } from '../../list-option';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import {
+    createEventListener,
+    waitAnimationFrame
+} from '../../utilities/tests/component';
+
+/**
+ * Page object for the `nimble-combobox` component to provide consistent ways
+ * of querying and interacting with the component during tests.
+ */
+export class ComboboxPageObject {
+    private readonly regionLoadedListener = createEventListener(
+        this.comboboxElement,
+        'loaded'
+    );
+
+    public constructor(protected readonly comboboxElement: Combobox) {}
+
+    /**
+     * Gets the selectable options in the drop-down. Does not include options that are disabled or filtered out.
+     */
+    public getOptions(): string[] {
```

Comment:
nit: I'd vote for a name that disambiguates this from a (not currently existent, but possibly desired in future) method that returns all the child options, including the ones that are filtered out). I'd be fine with `getFilteredOptions()`, `getAvailableOptions()`, `getSelectableOptions()`, etc.